x/concentrated-liquidity/tick.go

```diff
@@ -393,6 +396,11 @@ func (k Keeper) GetTickLiquidityNetInDirection(ctx sdk.Context, poolId uint64, t
 	startTickKey := types.TickIndexToBytes(startTick)
 	boundTickKey := types.TickIndexToBytes(boundTick.Int64())
 
+	startTickLiquidity = sdk.ZeroDec()
+	if startTick == p.GetCurrentTick() {
+		startTickLiquidity = p.GetLiquidity()
```

Comment:
Question: What is the reason we directly use pool's liquidity, not the pool's current tick's liquidity net?

Reply:
GetLiquidity should be analogous to getting the pools current tick liquidity.

If this is not the case, then further logic (unrelated to this PR) is incorrect. See https://github.com/osmosis-labs/osmosis/blob/04dae337c39555c8cab327e0e9f674b76e42378b/x/concentrated-liquidity/incentives.go#L404-L406

Would like confirmation that what I said is correct prior to merge.